package-overrides/npm/dot-prop@4.0.0.json

```diff
@@ -0,0 +1,3 @@
+{
+  "format": "es6"
```

Comment:
Ah I see it isn't using modules, just ES features, so just the change to `format: 'esm'` should be fine here.